test/ex_doc/formatter/html/autolink_test.exs

```diff
@@ -95,21 +92,20 @@ defmodule ExDoc.Formatter.HTML.AutolinkTest do
       assert Autolink.elixir_functions("`MyModule.example/2`", ["MyModule.example/2"]) ==
                "[`MyModule.example/2`](MyModule.html#example/2)"
 
-      assert Autolink.elixir_functions("`MyModule.Nested.example/2`", ["MyModule.Nested.example/2"]) ==
-               "[`MyModule.Nested.example/2`](MyModule.Nested.html#example/2)"
+      assert Autolink.elixir_functions("`MyModule.Nested.example/2`", [
+               "MyModule.Nested.example/2"
+             ]) == "[`MyModule.Nested.example/2`](MyModule.Nested.html#example/2)"
 
-      assert Autolink.elixir_functions(
-               "`Mod.example/2` then `Mod.example/2`",
-               ["Mod.example/2"]
-             ) == "[`Mod.example/2`](Mod.html#example/2) then [`Mod.example/2`](Mod.html#example/2)"
+      assert Autolink.elixir_functions("`Mod.example/2` then `Mod.example/2`", ["Mod.example/2"]) ==
+               "[`Mod.example/2`](Mod.html#example/2) then [`Mod.example/2`](Mod.html#example/2)"
 
       assert Autolink.elixir_functions("`  MyModule.spaces/0  `", ["MyModule.spaces/0"]) ==
                "[`MyModule.spaces/0`](MyModule.html#spaces/0)"
 
-      assert Autolink.elixir_functions(
-               "`ModA.example/1` and `ModB.example/2`",
-               ["ModA.example/1", "ModB.example/2"]
-             ) ==
+      assert Autolink.elixir_functions("`ModA.example/1` and `ModB.example/2`", [
+               "ModA.example/1",
+               "ModB.example/2"
+             ]) ==
```

Comment:
Can we please keep this example and previous ones in the previous syntax? I.e. one argument per line with the whole list on a single line? Thanks!